content/en/observability_pipelines/setup_opw.md

```diff
@@ -62,10 +62,22 @@ To set bootstrap options, do one of the following:
 : env var: `DD_OP_THREADS`
 : The number of threads to use for processing (optional, default: the number of available cores).
 
+`proxy`
+: env variables: `DD_HTTP_PROXY`, `DD_HTTPS_PROXY`
+: Set proxy servers for the Observability Pipelines Worker. The proxy configuration for the Worker works in the same way as it does for the [Datadog Agent][4].
```

Comment:
I think it would be helpful to point out to users that the DD_HTTP(s)_proxy and HTTP(s)_ proxy  environment variables are not ones that should be prepended to the installation script. In order for the worker to resolve these variables, they need to already be exported in the customer's environment.

Reply:
@neuronull added a note in this [commit](https://github.com/DataDog/documentation/pull/23902/commits/9ee5cfa303027aeb03072376af68c88de4128557).